### PR TITLE
Remove outdated builtin editors list...

### PR DESCRIPTION
**Pull request by mathieutu**
... and replace by the the dedicated page url.

### CODE REVIEW

File: docs/API Documentation.md [hunk 1]
```diff
@@ -367,11 +367,8 @@ PrettyPageHandler::addResourcePath(string $resourcesPath)
 // Sets an editor to use to open referenced files, either by
 // a string identifier, or as an arbitrary callable that returns
 // a string or an array that can be used as an href attribute.
-// Available built-in editors are:
-// - sublime
-// - emacs
-// - textmate
-// - macvim
+// Available built-in editors are available here: https://github.com/filp/whoops/blob/master/docs/Open Files In An Editor.md
```
Comment (staabm) [inline docs/API Documentation.md]: 2x availabe

Comment (mathieutu) [inline docs/API Documentation.md]: oh. :man_facepalming:

**Comment by denis-sokolov** [comment]
Thank you for your contribution!

**Comment by mathieutu** [comment]
My pleasure! ☺️

**Pull request merged**